content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,211 @@
+
+This technical specification outlines a series of security best practices that, if followed, enable Cerner to provide more detailed information about your application to end users. The goal of these best practices is to encourage the use of existing internet security features and standards. In particular, we want users to:
+
+- be assured that their data is being securely transmitted using modern transport layer security and encryption protocols
+- understand what legal entity they're sharing their data with
+
+These best practices should be considered as strongly-recommended prerequisites that software developers address during the application registration process, prior to making their applications available for general use.  Failure to comply with these prerequisites will result in Cerner describing your application as an "Unknown Application" to users during the OAuth authorization workflow.
+
+### Glossary of Terms
+
+* **eTLD** - Effective Top Level Domain is the base domain. eTLD values are defined in the [Public Suffix List](https://publicsuffix.org/list/public_suffix_list.dat).  Examples include: .com, .edu, .org, and .co.uk.
+* **eTLD+1** - "Effective top level domain plus 1" describes a domain name that is one level deeper than the effective top level domain (eTLD). Examples include examplehealth.com, cerner.com and cerner.co.uk.
+* **OV** - Organization Validated - a certificate issuance process that involves the certificate authority validating the organization (business entity) requesting the certificate.
+* **EV** - Extended Validation - a certificate issuance process that involves the certificate authority performing extended validation of the organization (business entity) requesting the certificate.
+* **DV** - Domain Validated - a certificate issuance process that involves the certificate authority validating the requester owns the domain. No validation of the business entity is performed.
+* **TLS** - Transport Layer Security
+* **DNS TXT record** - a "text" record in the Domain Name System (DNS).  TXT records usually take form of name=value.
+
+### Technical Requirements
+
+The following technical requirements are stated using [RFC-2119](https://datatracker.ietf.org/doc/html/rfc2119) terminology. These requirements focus on the security best practices that Cerner expects application developers to adhere to in order for Cerner to provide the best description of your application to end users:
+
+* Web applications MUST support the HTTPS 1.1 protocol.
+* Application SHOULD support TLS.  (Please refer to the [Cerner CODE Program's TLS guidance document](http://fhir.cerner.com/authorization/tls-guidance/) for further detailed practices related to TLS.)
+* Application SHOULD use (only) port 443 for TLS.
+* Application SHOULD support and prefer TLS 1.2 or higher.
+* Application SHOULD have a valid, non-expired TLS certificate issued from a [trusted authority](http://fhir.cerner.com/authorization/tls-guidance/).
```

Comment:
Are we able to put a TOC in the TLS guidance to refer to the specific section about trusted authorities? And maybe a TOC in this page?

Reply:
Added link directly to CA guidance in TLS Guidance doc. I did not add a TOC to the TLS Guidance but did add a rudimentary TOC to this page.